### PR TITLE
bem-forum: After submit commentaries from the preview-mode, previews remains hanging

### PR DESCRIPTION
**Pull request by gela-d**

### CODE REVIEW

**Comment by gela-d** [comment]
@tavriaforever

File: desktop.blocks/comments/comments.js [hunk 1]
```diff
@@ -162,6 +162,7 @@ modules.define('comments', ['i-bem__dom', 'jquery'], function (provide, BEMDOM,
             // подписываемся на удаление добавленного комментария
             this._subscribeDelete(this.findBlocksInside(this.findElem('item'), 'comment').pop());
 
+            this._form.findBlockInside('preview').domElem.html('');
```
Comment (tadatuta) [inline desktop.blocks/comments/comments.js]: I'd suggest to provide special method in `preview` to clear itself.
And you may cache `preview` for next usage (still not sure it's really useful).

Comment (gela-d) [inline desktop.blocks/comments/comments.js]: > I'd suggest to provide special method in preview to clear itself.

Ok, I'll do it.

> And you may cache preview for next usage (still not sure it's really useful).

Nooo. This case works only when we submit our new comment. What for we need cache it?

Comment (tadatuta) [inline desktop.blocks/comments/comments.js]: agree. if for the next comment there will be a new `preview` block generated — we don't need it.

**Comment by gela-d** [comment]
@tadatuta https://github.com/bem/bem-forum/commit/762499e379f173e2572089968aeab3342bb2e8a9

**Comment by gela-d** [comment]
> agree. if for the next comment there will be a new preview block generated — we don't need it.

I checked - in next comment everything is ok

**Comment by tadatuta** [comment]
lgtm

**Pull request merged**